app/viewmodels/checkAnswers/CountryOfOriginSummary.scala

```diff
@@ -46,7 +46,7 @@ object CountryOfOriginSummary {
       key = "countryOfOrigin.checkYourAnswersLabel",
       value = ValueViewModel(HtmlFormat.escape(value).toString),
       actions = Seq(
-        ActionItemViewModel("site.change", routes.CountryOfOriginController.onPageLoadUpdate(mode, recordId).url)
+        ActionItemViewModel("site.change", routes.HasCountryOfOriginChangeController.onPageLoad(mode, recordId).url)
```

Comment:
i am not sure about this - because this means that when the user clicks on the change link when they are checking the answers it goes right back to the warning page when i think i should go to the country of origin page -> you might need to check with amaka on this

Reply:
the right behavior is for it to go back to the country of origin page. Amaka confirmed it